### DISCUSSION TITLE
Boilerplate 🥗

### DISCUSSION THREAD

**Discussion by honaseung** [honaseung/react-deep-dive#16] [Terms]
***Bolierplate means common syntaxes.*** Which wouldn't be changed. Even if it has to be change, Changing it minimal. And use it essentially. For example, In JS, If you don't return specific value in function then you don't type `return` or `return undefined`. But after you call it, you can see `undefined`. Because it's a kind of 'Boilerplate'. Even if you don't type it, It will do. You can see more Example from [Here](https://ko.wikipedia.org/wiki/%EC%83%81%EC%9A%A9%EA%B5%AC_%EC%BD%94%EB%93%9C).

![boilerplate](https://github.com/honaseung/react-deep-dive/assets/53066347/13367603-b26b-4fdf-9989-880b9b4bd181)
<sup>I don't know what it is. I just searched an image for boilerplate. Looks too old and boilerplate codes are too. Maybe this  is the reason why it is called boilerplate.</sup>

**Comment by chaminju927** [comment]
I found additional meaning of boiler plate, which is 'reusable without change'! it would be nice to add this on your discussion !
